modules/sre-cluster-access.adoc

```diff
@@ -10,20 +12,277 @@ SRE access to {product-title}
 ifdef::openshift-rosa[]
 (ROSA)
 endif::openshift-rosa[]
-clusters is controlled through several layers of required authentication, all of which are managed by strict company policy. All authentication attempts to access a cluster, as well as changes made within a cluster, are recorded within audit logs, along with the specific account identity of the SRE responsible for those actions. These audit logs help ensure that all changes made by SREs to a customer's cluster adhere to the strict policies and procedures that make up Red Hat's managed services guidelines.
+clusters is controlled through several layers of required authentication, all of which are managed by strict company policy. All authentication attempts to access a cluster and changes made within a cluster are recorded within audit logs, along with the specific account identity of the SRE responsible for those actions. These audit logs help ensure that all changes made by SREs to a customer's cluster adhere to the strict policies and procedures that make up Red Hat's managed services guidelines.
```

Comment:
"Clusters are controlled..."

Reply:
Going to leave this one "as is" because the full sentence is "SRE access to clusters is..."